lib/args.js

```diff
@@ -36,7 +37,7 @@ function args(argv) {
       'mocha-path'],
     boolean: ['help', 'version', 'watch', 'cover', 'node', 'wd', 'debug',
       'invert', 'recursive', 'colors', 'ignore-ssl-errors', 'browser-field',
-      'commondir', 'allow-chrome-as-root', 'async-polling'],
+      'commondir', 'allow-chrome-as-root', 'async-polling', 'dumpio'],
```

Comment:
Should this print a warning (or even fail) when being passed together with `--node`?

Reply:
I think we don't fail on invalid argument combinations at all yet, do we?

Reply:
I think we do here: https://github.com/mantoni/mochify.js/blob/90886285fea8ec819ce622861976c57bd70103ad/lib/mochify.js#L70-L79 ?

Reply:
Yeah, right. But in general, there are heaps of invalid / useless combinations, like, all chromium specific options with `--wd` or `--consolify` ... If we want to do this, I'd say it's a separate change introducing some sort of valid option mapping.

Reply:
Yeah, I agree. Let's not do this here.